default.py

```diff
@@ -465,16 +511,16 @@ def onClick(self, controlId):  # pylint: disable=invalid-name
                                 #coachGui.doModal()
                                 #del coachGui
                             if game_version == 'condensed':
-                                game_streams = gp.get_stream(condensed_id, 'game', username=username)
+                                stream_url = self.select_stream_url(gp.get_stream(condensed_id, 'game', username=username))
+                            elif game_version == 'coach':
+                                stream_url = self.select_stream_url(gp.get_stream(coach_id, 'game', username=username))
                             else:
-                                if game_version == 'coach':
-                                    game_streams = gp.get_stream(coach_id, 'game', username=username)
-                                else:
-                                    game_streams = gp.get_stream(video_id, 'game', username=username)
-                            bitrate = self.select_bitrate(game_streams.keys())
-                            if bitrate:
-                                game_url = game_streams[bitrate]
-                                self.play_url(game_url)
+                                stream_url = self.select_stream_url(gp.get_stream(video_id, 'game', username=username))
+                            if stream_url:
+                                self.play_url(stream_url)
+                            elif stream_url is False:
```

Comment:
@emilsvennesson Ahh, nice.

Ok, this looks good and will merge soon.

---Alex